### ISSUE TITLE
Paypal Button renders then immediately disappears

### ISSUE THREAD

**Issue by tomasmd** [paypal/paypal-checkout#984]
I installed the paypal express plugin [for my woocommerce site:](https://dev.cmhelmets.com/product/sebastian-vettel-2018-montecarlo-monaco-gp-f1-replica-helmet-full-size/)

You can reproduce it by following that url and quickly scrolling down to the "add to cart" button. The paypal buttons appear and disappear just below it in less than a second, once the page is finished loading.

I believe this might be the same issue as [#416](https://github.com/paypal/paypal-checkout/issues/416)
However, I haven't been able to gather any information using the break on method that is suggested there.
If i could isolate the function that is causing the button to disappear, I would be able to prevent it from occuring. 

It'd be a big help if you could help me debug this

**Comment by mcntsh** [comment]
Hey @tomasmd, I just took a quick look and it seems like the page is loaded and PayPal Checkout is rendered, then the page container is [removed and re-appended in `MaterialLayout.prototype.init`](https://dev.cmhelmets.com/wp-content/themes/fab//assets/js/theme.js?ver=1.10.4) without re-initializing PayPal Checkout.

**Comment by tomasmd** [comment]
Hi Jon.

Thanks for the help.
I'll see if I can re initialize payPal once it's loaded into the new
container

Regards,




* Tomás Viera*
IT Developer
Aguirre 1551 Piso 3 Of. 3
C1414ATE - C.A.B.A., Argentina
Tel: +54 11 4855 7899



<https://www.facebook.com/md.marketingdigital>
<https://twitter.com/MarketingD_Arg> <https://twitter.com/MarketingD_Arg>
<https://plus.google.com/+Mdmarketingdigital>
<https://plus.google.com/113962109400250039622/>
<http://www.linkedin.com/company/md-marketing-digital>
<https://www.linkedin.com/company/md-marketing-digital>
<https://es.pinterest.com/mdigital/>
<https://www.instagram.com/md.marketing.digital/>

**Comment by mcntsh** [comment]
Hey @laphilia, please see this issue: https://github.com/paypal/paypal-checkout/issues/1006. 

@tomasmd since we haven't heard back in a while, I'm going to close this issue out. If you have any updates on your side that need our eyes, feel free to open up a new ticket :).
